tests/foreman/ui/test_ansible.py

```diff
@@ -726,9 +726,11 @@ def test_positive_ansible_job_check_mode(self):
         :CaseAutomation: NotAutomated
         """
 
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_install_ansible_collection_via_job_invocation(self):
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([8])
```

Comment:
```suggestion
  @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Reply:
Please use `settings.content_host.default_rhel_version` or parameterize it for 8 and 9

Reply:
We've many ansible tests using the hardcoded version, so I'd keep this as is for now, and we can change it later along with other tests

Reply:
Updated this for now